### PR TITLE
Use basisregisters sqs

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Requests/Building/BuildingLambdaRequest.cs [hunk 1]
```diff
@@ -1,30 +1,17 @@
 namespace BuildingRegistry.Api.BackOffice.Handlers.Lambda.Requests.Building
 {
-    using System;
-    using System.Collections.Generic;
     using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
-    using MediatR;
+    using Be.Vlaanderen.Basisregisters.Sqs.Lambda.Requests;
 
-    public abstract class BuildingLambdaRequest : IRequest
+    public record BuildingLambdaRequest : SqsLambdaRequest
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Requests/Building/BuildingLambdaRequest.cs:6]: seal all records

Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Requests/Building/BuildingLambdaRequest.cs:6]: abstract record?

Comment (jvandaal) [reply src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Requests/Building/BuildingLambdaRequest.cs]: Changed

File: src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/BuildingUnit/BuildingUnitLambdaHandler.cs [hunk 2]
```diff
@@ -1,39 +1,33 @@
...
+    public abstract class BuildingUnitLambdaHandler<TSqsLambdaRequest> : SqsLambdaHandlerBase<TSqsLambdaRequest>
         where TSqsLambdaRequest : BuildingUnitLambdaRequest
     {
-        private readonly ITicketing _ticketing;
-        private readonly ICustomRetryPolicy _retryPolicy;
         private readonly IBuildings _buildings;
 
-        protected IIdempotentCommandHandler IdempotentCommandHandler { get; }
-        protected string DetailUrlFormat { get; }
+        protected readonly string DetailUrlFormat;
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/BuildingUnit/BuildingUnitLambdaHandler.cs:21]: convert to property

File: src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/Building/BuildingLambdaHandler.cs [hunk 3]
```diff
@@ -1,39 +1,33 @@
...
+    public abstract class BuildingLambdaHandler<TSqsLambdaRequest> : SqsLambdaHandlerBase<TSqsLambdaRequest>
         where TSqsLambdaRequest : BuildingLambdaRequest
     {
-        private readonly ITicketing _ticketing;
-        private readonly ICustomRetryPolicy _retryPolicy;
         private readonly IBuildings _buildings;
 
-        protected IIdempotentCommandHandler IdempotentCommandHandler { get; }
-        protected string DetailUrlFormat { get; }
+        protected readonly string DetailUrlFormat;
```
Comment (ArneD) [inline src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/Building/BuildingLambdaHandler.cs:21]: Convert to property

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/BuildingUnit/BuildingUnitLambdaHandler.cs (hunk 2 above)
Comment (jvandaal) [reply src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Handlers/BuildingUnit/BuildingUnitLambdaHandler.cs]: Changed

**Pull request merged**